x/exchange/spec/01_concepts.md

```diff
@@ -0,0 +1,336 @@
+
+* An order is settling for `400chicken`: `400 * 3 / 1000` = `1.2`, which is rounded up to `2chicken`.
+* An order is settling for `3000chicken`: `3000 * 3 / 1000` = `9`, which doesn't need rounding, so stays at `9chicken`.
+
+The actual amount isn't known until settlement, but a minimum can be calculated by applying the applicable ratio to an ask order's `price`.
+The seller settlement ratio fee will be at least that amount, but since it gets larger slower than the price, `<ask order price> - <ratio fee based on ask order price> - <flat fee>` is the least amount the seller will end up with.
+
+
+#### Buyer Settlement Ratio Fee
+
+A market's `fee_buyer_settlement_ratios` can have `FeeRatios` with any denom pair, i.e. the `price` and `fee` do not need to be the same denom.
+It can also have multiple entries with the same `price` denom or `fee` denom, but it can only have one entry for each `price` to `fee` denom pair.
+E.g. a market can have `100chicken:1cow` and also `100chicken:7chicken`, `500cow:1cow`, and `5cow:1chicken`, but it couldn't also have `105chicken:2cow`.
+
+To calculate the buyer settlement ratio fee, the following formula is used: `<bid price> * <ratio fee> / <ratio price>`.
+If that is not a whole number, the chosen ratio is not applicable to the bid order's price and cannot be used.
+The user will need to either use a different ratio or change their bid price.
+
+The buyer settlement ratio fee should be added to the buyer settlement flat fee and provided in the `buyer_settlement_fees` in the bid order.
+The ratio and flat fees can be in any denoms allowed by the market, and do not have to be the same.
+
+
+### Exchange Fees
+
+A portion of the fees collected by a market, are given to the exchange.
+The amount is defined using basis points in the exchange module's [Params](06_params.md#params) and can be configured differently for specific denoms.
+
+When the market collects fees, the applicable basis points are looked up and applied to the amount being collected.
+That amount is then transferred from the market's account to the chain's fee collector (similar to gas fees).
+
+The following formula is used for each denom in the fees being collected: `<fee amount> * <basis points> / 10,000`.
+If that is not a whole number, it is rounded up to the next whole number.
+
+For example, Say the exchange has a default split of `500` (basis points), and a specific split of `100` for `rooster`.
+When a market collects a fee of `1500hen,710rooster`:
+There is no specific split for `hen`, so the default `500` is used for them. `1500 * 500 / 10,000` = `75hen` (a whole number, so no rounding is needed).
+The specific `rooster` split of `100` is used for those: `710 * 100 / 10,000` = `7.1` which gets rounded up to `8rooster`.
+So the market will first receive `1500hen,710rooster` from the buyer(s)/seller(s), then `75hen,8rooster` is transferred from the market to the fee collector.
```

Comment:
When we do the larger fee system refactor we will have to target any captured fees that are not in the chain "feedenom" and transform them into the feedenom... using another market...